client/me/purchases/manage-purchase/notices.jsx

```diff
@@ -930,6 +945,11 @@ class PurchaseNotice extends Component {
 			return null;
 		}
 
+		const nonProductOwnerNotice = this.renderNonProductOwnerNotice();
+		if ( ! this.props.isProductOwner ) {
+			return nonProductOwnerNotice;
```

Comment:
It's minor, but if we change this to `return this.renderNonProductOwnerNotice()` then it will prevent rendering all of that notice if `isProductOwner` is falsey instead of on every render (even though it's being ignored if that prop is unset).

Reply:
oh, that makes sense! I'll update it.